Domain/Aggreagtes/AdminAggregate/Admin.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Domain.Aggreagtes.UserAggregate;
+using Domain.Common.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Aggreagtes.AdminAggregate
+{
+    public class Admin : AuditableEntity, IAggregateRoot
```

Comment:
Shouldn't we rename this StaffAggregate.? We can then use role to indicate those that are admins?

Reply:
this is be solve on this branch 
https://github.com/sysbeams/codelearnersmanagmentapp/pull/70

Reply:
we have a staff entity already 
https://github.com/sysbeams/codelearnersmanagmentapp/pull/67/files